clearpath_generator_common/clearpath_generator_common/param/manipulators.py

```diff
@@ -78,9 +78,10 @@ def generate_parameters(self, use_sim_time: bool = False) -> None:
                 package=self.default_parameter_package,
                 path=self.default_parameter_directory,
             )
+            namespace = self.namespace if self.namespace != '/' else ''
```

Comment:
For readability, can we please expand this into a normal if/else, rather than using Python's ternary?